src/utils/keyboardCopy.ts

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+enum OperatingSystems {
+  Mac = 'MacOS',
+  Windows = 'Windows',
+  Linux = 'Linux',
+}
+
+export const keyboardCopyTriggered = (
```

Comment:
this can just stay in the `crossPlatform` file I think, since they both share the `OperatingSystems` enum and they're both focused on cross platform operations.